source/install/desktop-app-changelog.rst

```diff
@@ -100,6 +182,7 @@ All Platforms
 Known Issues
```

Comment:
We should probably add a known issue for this: https://community.mattermost.com/core/pl/hhy4tn6ot7r8ffx18ywaofuxhw

Reply:
I just added it 👍

Reply:
Somehow I didn't see it my bad :D